### ISSUE TITLE
Invalid file: "master.php"

### ISSUE THREAD

**Issue by carlkyo** [lonnieezell/Bonfire2#352]
encounter error of Invalid file: "master.php" after  login with 2fa
Invalid file: "master.php"

**Comment by daracorcoran** [comment]
@lonnieezell would Bonfire need to extend the ActionContoller and Action classes to fix this?

**Comment by lonnieezell** [comment]
Yes, and extend their view methods similarly to how we already extend the LoginController, RegisterController and MagicLinkControllers. I was just noticing this morning that those weren't extended and that we're using Shield views for some of the actions, not Bonfire views so there's definitely more to do around that area.
